alphagov/paas-cf
# [#163221178] Update buildpacks to latest

BLOCKED: Need to email tenants to inform them of the pending upgrade.


What
----
BLOCKED
Updating the buildpacks can now be done with `update_buildpacks.sh`.

This script runs go code at `tools/buildpacks` which:

1. Parses the buildpacks.yml file
2. Contacts the GitHub API to find the latest version for each of the configured buildpacks
3. Updates buildpacks.yml with these latest versions

Buildpacks were then updated to the latest versions, and committed. 

How to review
-------------

* Code Review
* Test Go code:
    1. Run `scripts/update_buildpacks.sh` and ensure there are no differences
* Test generated buildpack config:
    1. Merge this branch with update-buildpacks-yaml-config **_in a new branch_** and push
    1. Deploy to your environment, and ensure that the buildpacks are updated to the correct version (comparing them to the yaml file / latest releases on cloudfoundry's GitHub) (`cf buildpacks`)

Who can review
--------------

Anyone but @tnwhitwell @richardTowers
